mappings/net/minecraft/TerrainFeatures.mapping

```diff
@@ -0,0 +1,86 @@
+CLASS net/minecraft/class_5464 net/minecraft/TerrainFeatures
```

Comment:
Should be renamed to `net/minecraft/world/gen/feature/ConfiguredFeatures`

Reply:
definitely not have them stashed in top level net/minecraft but in /world/gen/features like liach suggests.